src/main/io/displayport_msp_bf_compat.c

```diff
@@ -58,42 +58,44 @@ uint8_t getBfCharacter(uint8_t ch, uint8_t page)
     case SYM_DEGREES:
         return BF_SYM_GPS_DEGREE;
 
+/*
     case SYM_HEADING:
-        return BF_SYM_OVER_HOME;
+        return BF_SYM_HEADING;
+
+    case SYM_SCALE:
+        return BF_SYM_SCALE;
 
-/*    case SYM_SCALE:
-        return BF_SYM_SCALE; 
-*/
     case SYM_HDP_L:
-        return 'H';
+        return BF_SYM_HDP_L;
 
     case SYM_HDP_R:
-        return 'P';     // The idea is to form HP, resembling "HDOP"
-
+        return BF_SYM_HDP_R;
+*/
     case SYM_HOME:
         return BF_SYM_HOMEFLAG;
 
     case SYM_2RSS:
         return BF_SYM_RSSI;
 
+/*
     case SYM_DB:
-        return 'D';     // Just D to resemble "dB"
+        return BF_SYM_DB
 
     case SYM_DBM:
-        return BF_SYM_BLANK;    // No dbm resembling option
+        return BF_SYM_DBM;
 
-/*    case SYM_SNR:
+    case SYM_SNR:
         return BF_SYM_SNR;
 
     case SYM_AH_DECORATION_UP:
-        return BF_SYM_AH_DECORATION;
+        return BF_SYM_AH_DECORATION_UP;
 
     case SYM_AH_DECORATION_DOWN:
-        return BF_SYM_AH_DECORATION;
-*/
-    case SYM_DIRECTION:
```

Comment:
Same as my previous comment. It might be intended as a house, but visually, it's a perfect arrowhead. And since the objective here is visual representation, I believe this fits perfectly the usecase.

![IMG_6448](https://user-images.githubusercontent.com/9812730/228905986-660c6a91-f2d2-4e64-b789-99f090c24a91.JPG)